utilities.c

```diff
@@ -359,3 +365,25 @@ ssize_t __iio_printf iio_snprintf(char *buf, size_t len, const char *fmt, ...)
 
 	return (ssize_t)ret;
 }
+
+uint64_t iio_read_counter_us(void)
+{
+	uint64_t value;
+
+#ifdef _WIN32
+	LARGE_INTEGER freq, cnt;
+
+	QueryPerformanceFrequency(&freq);
+	QueryPerformanceCounter(&cnt);
+
+	value = (1000000 * cnt.QuadPart) / freq.QuadPart;
+#else
+	struct timeval tv;
+
+	gettimeofday(&tv, NULL);
```

Comment:
from the man page:

```
APPLICATION USAGE
       Applications should use the clock_gettime() function instead of the obsolescent gettimeofday() function.

RATIONALE
       None.

FUTURE DIRECTIONS
       The gettimeofday() function may be removed in a future version.

```

Reply:
I had no idea. I updated the code for clock_gettime() with the CLOCK_MONOTONIC clock now.